cmd/otelcol/fips/config/gateway_config.yaml

```
@@ -27,8 +32,12 @@ receivers:
     protocols:
       grpc:
         endpoint: "${SPLUNK_LISTEN_INTERFACE}:4317"
+        # Whether to preserve incoming access token and use instead of exporter token
```

```suggestion
        # Uncomment below config to preserve incoming access token and use it instead of the token value set in exporter config
```
just a tiny change to the wording to make it more clear for a user

---

Thanks! Agree! I used existing comment, updating all files...